src/components/ceremony/handlers/create-engagement-default-ceremony.handler.ts

```diff
@@ -0,0 +1,43 @@
+import { node, relation } from 'cypher-query-builder';
+import { DateTime } from 'luxon';
+import { ServerException } from '../../../common';
+import { DatabaseService, EventsHandler, IEventHandler } from '../../../core';
+import { EngagementCreatedEvent } from '../../engagement/events';
+import { CeremonyService } from '../ceremony.service';
+import { CeremonyType } from '../dto/type.enum';
+
+@EventsHandler(EngagementCreatedEvent)
+export class CreateEngagementDefaultCeremonyHandler
+  implements IEventHandler<EngagementCreatedEvent> {
+  constructor(
+    private readonly ceremonies: CeremonyService,
+    private readonly db: DatabaseService
+  ) {}
+
+  async handle({ engagement, session }: EngagementCreatedEvent) {
+    const input = { type: CeremonyType.Certification };
```

Comment:
What about language engagements? They get the dedication type, not certification.

Reply:
I updated CeremonyType based on engagement typename.